SEED-platform/seed
# 2543 feat/maintenance page

#### Any background context you want to provide?
Needed a more graceful way to let users know the app is down
#### What's this PR do?
Creates a custom 503 service unavailable page for Nginx to serve 
#### How should this be manually tested?
Checkout this branch and run the docker workflow for a local deployment.
`docker-compose build`
`docker-compose up`
in another terminal tab:
`docker exec -it <docker_hash-of-web-container> bash`
`./docker/maintenance.sh on`
In your browser - open `localhost` and it should display the custom page
back in the docker container 
`./docker/maintenance.sh off` 
in your browser - this should now resolve to the landing page on refresh.  
#### What are the relevant tickets?
 #2543 
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/31779424/104606604-2e9d8100-563d-11eb-8858-75adae622f5f.png)
